presto-cli/src/main/java/com/facebook/presto/cli/ErrorMessages.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cli;
+
+import com.facebook.presto.client.ClientSession;
+
+import java.util.Arrays;
+
+public class ErrorMessages
+{
+    private ErrorMessages()
+    {}
+
+    public static String createErrorMessage(Throwable throwable, ClientSession session)
+    {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Error running command: " + throwable.getMessage());
+        if (session.isDebug()) {
+            builder.append(getStackTraceString(throwable));
+        }
+        return builder.toString();
+    }
+
+    private static String getStackTraceString(Throwable throwable)
+    {
+        StringBuilder builder = new StringBuilder();
+        builder.append(Arrays.stream(throwable.getStackTrace()).map(x -> x.toString()).reduce("", (x, y) -> x + y + "\n"));
+        if (throwable.getCause() != null) {
+            builder.append("caused by:\n");
+            builder.append(getStackTraceString(throwable.getCause()));
+        }
+        return builder.toString();
```

Comment:
This was just a leftover, in new version I don't implement printing stack by myself.
I change this commit to get rid of this leftover, so it should not be visible in any commit now.